lib/appliance_console/service_group.rb

```diff
@@ -41,32 +28,36 @@ def restart
     end
 
     def enable
-      LinuxAdmin.run("chkconfig", :params => {"--add" => "miqtop"})  # Is this really needed?
-      service_command("enable")
+      enable_miqtop
+      SERVICES.each { |s| run_service(s, "enable") }
+      run_service(POSTGRESQL_SERVICE, "enable") if postgresql?
     end
 
     def disable
-      service_command("disable")
+      run_service(POSTGRESQL_SERVICE, "disable") unless postgresql?
     end
 
     def start
-      start_command
+      SERVICES.each { |s| run_detached_service(s, "start") }
```

Comment:
100%

This whole refactor is because I was trying to move detach into linux admin - and introduced a bug.

(well that and there was a hidden bug when calling a service that had a bad name and got no feedback ...)